docs/csharp/language-reference/language-specification/index.md

```diff
@@ -1,29 +0,0 @@
----
-title: "C# 6.0 draft language specification"
-ms.date: 05/22/2018
-ms.topic: language-reference
-helpviewer_keywords: 
-  - "C# language, specification"
-  - "what's new [C#], language specification"
-  - "Visual C#, C# language specification"
-  - "language specification [C#]"
-ms.assetid: e5d5a5cc-636b-4bff-b9c8-a8edc6207c22
----
-# C# 6.0 draft language specification
-
-The C# language specification is the definitive source for C# syntax and usage. This specification contains detailed information about all aspects of the language, including many points that the documentation for C# doesn't cover.
-
-Version 5.0 of the specification has been released in December 2017 as the [Standard ECMA-334 5th Edition](https://www.ecma-international.org/publications/files/ECMA-ST/ECMA-334.pdf) document.
```

Comment:
With this removal, the link to the official latest spec is gone. Or that's fine, because the C# 6 draft includes the related content?

Reply:
I agree. I moved this text into the language reference index page.